packages/jest-mock/src/index.ts

```diff
@@ -1146,7 +1146,7 @@ export class ModuleMocker {
   ): MockInstance {
     if (typeof object !== 'object' && typeof object !== 'function') {
       throw new Error(
-        `Cannot spyOn on a primitive value; ${this._typeOf(object)} given`,
+        `Cannot spyOn a primitive value; ${this._typeOf(object)} given`,
```

Comment:
```suggestion
        `Cannot use spyOn on a primitive value; ${this._typeOf(object)} given`,
```

what about this? `spyOn` is a function, not a verb

Reply:
Good point!